scripts/update-dockerhub-description.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+image_name="all-in-one"
+readme_path="./README.md"
+token="$DOCKERHUB_TOKEN"
+repository="jaegertracing/$image_name"
+url="https://hub.docker.com/v2/repositories/$repository/"
+readme_content=$(cat $readme_path)
+payload="{\"full_description\":\"$readme_content\"}"
```

Comment:
L9 might cause a large output because of `set -x`. Instead of reading content in, you can create a temp file.

Also, inserting content of a file inside a JSON `"..."` string is very likely to result in malformed JSON.